source/agora/script/Engine.d

```diff
@@ -165,9 +165,9 @@ public class Engine
             if (!key.isValid())
                 return "LockType.Key 32-byte public key in lock script is invalid";
 
-            if (unlock.bytes.length != Signature.sizeof)
+            if (unlock.bytes.length != Schnorr.Signature.sizeof)
```

Comment:
I'd say just remove the rename of the import, it was only there to work around the name clash.

Reply:
I thought it might be to document but more likely because it was a clash.

Reply:
Updated.